backend/src/main/setup.py

```
@@ -18,13 +18,18 @@ async def start_setup(setup: Setup):
+        config['Ollama'] = {
+            'host': setup.OLLAMA_HOST
+        }
+
 
     # LAST STEP
     config['Global'] = {
         'FirstSetup': 'False',
         'BINDING_KEY': hash_key(setup.BINDING_KEY)
     }
-    with open('config.ini', 'w') as configfile:
+    with open('../../config.ini', 'w') as configfile:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)